notebooks/ImvizExample.ipynb

```diff
@@ -92,69 +102,61 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "acs_47tuc_1 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03gjq_flc.fits', cache=True)\n",
-    "acs_47tuc_2 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03h1q_flc.fits', cache=True)"
+    "files = ['jw02731-o001_t017_nircam_clear-f090w_i2d.fits',\n",
```

Comment:
Maybe tell people how big these files are. I am still waiting for the first file to download and I am not sure how long it is supposed to take. 👀

Reply:
To answer my own question (but should still be added to the notebook narrative, I think):

* `jw02731-o001_t017_nircam_clear-f090w_i2d.fits` 3.3GB
* `jw02731-o001_t017_nircam_f444w-f470n_i2d.fits` 789MB

In contrast, the HST/ACS files are around 161MB each (more or less).

Reply:
Thinking about this more, even though it will be less impressive, I wonder if we should stick to smaller subarrays in the example notebook... 💭